brands/amenity/bank.json

```
@@ -6214,6 +6229,19 @@
       "name:en": "VTB Bank"
     }
   },
+  "amenity/bank|ВТБ~Украина": {
+    "countryCodes": ["ua"],
+    "matchNames": ["втб банк"],
+    "tags": {
+      "amenity": "bank",
+      "brand": "ВТБ",
+      "brand:en": "VTB Bank",
+      "brand:wikidata": "Q16628820",
+      "brand:wikipedia": "ua:ВТБ#ПАТ_«ВТБ_Банк_(Україна)»",
```

For this one it looks like it actually links to a different Wikipedia page.  Also make sure to replace the underscores with spaces in the interwiki link
```suggestion
      "brand:wikipedia": "ru:ВТБ Банк (Украина)",
```

---

`ua` wiki does not have a sepparate page for this bank, just a paragraph in a page of a parent bank. I think a link to subsection should be parseble?

---

> I think a link to subsection should be parseble?

I think it is parsable (but replacing the underscores with spaces).  I would prefer things to have their own pages  - ideally someone would make a "ua:ВТБ Банк (Украина)".   But if you really like the section link with the "#" anchor, that's cool too.

Also, I don't really see these `brand:wikipedia` tags as all that important (or stable for that matter).  They are a convenience for humans.

As long as the more important `brand:wikidata` tag is there, people can follow that to view a wikipedia page in whatever languages are available.